alfresco-dynamic-extensions-repo/build.gradle

```diff
@@ -27,7 +27,9 @@ subprojects {
         ampLib(project(":annotations-runtime:scheduler-quartz-1")) { transitive = false }
         ampLib(project(":annotations-runtime:scheduler-quartz-2")) { transitive = false }
 
-        ampLib(project(":webscripts")) { transitive = false }
+        // Transitive dependencies are required to retain javax.annotations dependency
```

Comment:
It is not dynamic-extension's job to ship `javax.annotations` for all of Alfresco, and this may result in multiple identical classes on the classpath, and all problems that arise from it.

Reply:
@vierbergenlars can you propose a better solution?

Reply:
@todorinskiz this `javax.annotations` dependency is provided by Alfresco?

Reply:
I would be very surprised if nothing of Alfresco or Spring uses these annotations, and already has it as a dependency.